teamcity-test.sh

```diff
@@ -6,5 +6,6 @@ docker run \
     --workdir=/go/src/github.com/cockroachdb/loadgen \
     --volume="${GOPATH%%:*}/src":/go/src \
     --volume="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)":/go/src/github.com/cockroachdb/loadgen \
```

Comment:
can you tack on `:ro` here so errant writes fail the build instead of bricking the agent?